user/enterprise/tcie-3.x-setting-up-travis-ci-enterprise.md

```diff
@@ -0,0 +1,288 @@
+
+The Travis CI Enterprise Platform handles licensing, coordinates worker
+processes, and maintains the Enterprise user and admin dashboard. It must be
+installed as Kubernetes cluster or on at least one machine instance acting as microk8s host 
+or multiple instances managed by Kubernetes, separate from that of the Travis CI
+Enterprise worker. We recommend **compute optimized** instance running
+Ubuntu 18.04 LTS or later as the underlying operating system.
+
+> If you are migrating from Travis CI enteprise 2.x, please make sure to read and execute the [datbase migration instructions](/user/enterprise/tcie-3.x-migrating-db-from-2.x-to-3.x./) first.
+
+1. *On your infrastructure management platform*, create a Travis CI Platform Security Group.
+
+    If you're setting up your instance image or Kubernetes cluster for the first time, you most probably need to create some kind of
+    a Security Group. Create an entry for each port in the table below:
+
+    | Port | Service         | Description                                                                  |
+    |:-----|:----------------|:-----------------------------------------------------------------------------|
+    | 8800 | Custom TCP Rule | This port is to access the admin dashboard for your Enterprise installation. |
+    | 5672 | Custom TCP Rule | For RabbitMQ Non-SSL.                                                        |
+    | 4567 | Custom TCP Rule | For RabbitMQ SSL.                                                            |
+    | 3333 | HTTPS           | TCIE 3.x User administration Web application over HTTPS access.              |
+    | 443  | HTTPS           | Web application over HTTPS access.                                           |
+    | 80   | HTTP            | Web application access.                                                      |
+    | 22   | SSH             | SSH access.                                                                  |
+
+2. If you're using a hostname and not just an IP address, this would be a time to configure it.
+3. Set-up your cluster/machine instances configuration - at least 1 virtual machine is needed (in microk8s scenario) or Kubernetes cluster prepared and started
+4. On *your local machine*, assuming the proper connection with the cluster exists, install Replicated KOTS.
+5. TCIE 3.x installed on **single** virtual machine instance
+    1. On *your new virtual machine instance* run kurl.sh via 'curl https://kurl.sh/latest | sudo bash' 
+    2. Get credentials from microk8s cluster running on *your new vm instance*
+    3. Please refer to various cluster and installation options in [kURL documentation](https://kurl.sh/docs/install-with-kurl/). Please adjust your setup and configuration according to your needs before progressing with TCIE 3.X installation
+6. TCIE 3.x installed as Kubernetes cluster in the cloud
+    1. Connect to your cluster and get it's configuration, particularly credentials
```

Comment:
```suggestion
    1. Connect to your cluster and get its configuration, particularly credentials
```

How does this work? What command do I need to run to get credentials?

Reply:
Adressed by rephrasing the sentence to be more exact in subsequent commit